docs/shipping/Code/python.md

```diff
@@ -1402,7 +1632,7 @@ Typically, the service name will be `logzio-k8s-telemetry.default.svc.cluster.lo
   
 ```shell
 kubectl run -it --image=k8s.gcr.io/e2e-test-images/jessie-dnsutils:1.3 --restart=Never shell -- \
-sh -c 'nslookup kubernetes.default | grep Name | sed "s/Name:\skubernetes.default//"'
+shell -c 'nslookup kubernetes.default | grep Name | sed "s/Name:\skubernetes.default//"'
```

Comment:
I think it was accidentally changed - it should stay `sh` and not turn to `shell`